libcxx/test/libcxx/ranges/range.adaptors/range.stride.view/iterator/dereference.pass.cpp

```diff
@@ -0,0 +1,34 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+// REQUIRES: libcpp-hardening-mode={{fast|extensive|debug}}
+// XFAIL:libcpp-hardening-mode=debug && availability-verbose_abort-missing
+
+// constexpr decltype(auto) operator*() const
+
+#include "check_assertion.h"
+#include <ranges>
```

Comment:
```suggestion
#include <ranges>
#include <utility>

#include "check_assertion.h"
```

Two nits: we prefer library headers first, and this was missing `<utility>` for `as_const`.